src/mon/MDSMonitor.cc

```diff
@@ -2069,6 +2092,9 @@ bool MDSMonitor::maybe_promote_standby(std::shared_ptr<Filesystem> fs)
         const MDSMap::mds_info_t si = pending_fsmap.get_info_gid(sgid);
         dout(0) << " taking over failed mds." << f << " with " << sgid
                 << "/" << si.name << " " << si.addr << dendl;
+        mon->clog->info() << "Activating MDS " << si.name << " in filesystem "
+                          << fs->mds_map.fs_name << " as rank " << f;
```

Comment:
Agreed - I've changed to "Assigned standby MDS <name> to filesystem <fs> as  rank <n>"